ibc-core/ics04-channel/src/handler/acknowledgement.rs

```diff
@@ -74,11 +74,6 @@ where
 
     // apply state changes
     {
-        let commitment_path_on_a = CommitmentPath {
```

Comment:
Are these path removals as additional clean-ups to this PR?
Would you open up an issue and include it in the changelog?

Reply:
This came up from [`shadow_reuse`](https://rust-lang.github.io/rust-clippy/master/index.html#/shadow_reuse) or [`shadow_unrelated`](https://rust-lang.github.io/rust-clippy/master/index.html#/shadow_unrelated). I didn't add others - but this looked reasonable to be fixed.

Reply:
Thank you for the issue 👍🏻